src/components/RichTextEditor/serializer/markdown/remarkMention/types.ts

```diff
@@ -0,0 +1,21 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type NodeChild = Record<string, string | [] | NodeChild[]> & {
```

Comment:
Ho would you solve this problem?
`Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'NodeChild'.
  No index signature with a parameter of type 'string' was found on type 'NodeChild'`

Reply:
Updated